src/content/deployment/flavors-ios.md

```diff
@@ -0,0 +1,471 @@
+process from compilation and linking to debugging and
+distribution. One way that you can use build settings
+with Flutter flavors is to assign those build settings
+to Xcode build configurations. For example, you might want
+to assign different API URLs to  `Debug-staging` and
+`Debug-production`. For example: 
+
+```console title="debug-staging-settings.xcconfig"
+# Debug-staging build settings
+API_BASE_URL = staging.flavors.com/api
+```
+
+```console title="debug-production-settings.xcconfig"
+# Debug-production build settings
+API_BASE_URL = flavors.com/api
+```
+
+If you would like to add additional build settings for
+a specific build configuration, see Apple's
+[Adding a build configuration file to your project][].
+
+[build settings]: https://developer.apple.com/documentation/xcode/build-settings-reference/
+[Adding a build configuration file to your project]: https://developer.apple.com/documentation/xcode/adding-a-build-configuration-file-to-your-project
+
+### Add additional customizations
+
+This document contains a few common Xcode scheme
+configurations, but there are many more that you can apply.
+To learn about them, see
+[Customizing the build schemes for a project][].
+
+[Customizing the build schemes for a project]: https://developer.apple.com/documentation/xcode/customizing-the-build-schemes-for-a-project
+
+### Update Podfiles
+
+If you are creating new Xcode schemes for a Flutter iOS
+project and you have an iOS Podfile in an existing
+Flutter project, you must update the Flutter iOS Podfile to
+match the changes you made in Xcode.
+
+The following steps show how to update your iOS Podfile to
+include two new Xcode schemes called `staging` and
+`production` in a Flutter project called `flavors_example`.
+You can also use these steps to update a macOS
+project by replacing any reference to `iOS` with `macOS`.
+
+1. In your IDE, open the `ios/Podfile` file.
+1. Make the following updates and save your changes.
+
+    ```json title="flavors_example/ios/Podfile"
+    project 'Runner', {
+      ...
+      'Debug' => :debug,
+      'Debug-staging' => :debug,
+      'Debug-production' => :debug,
+      'Profile' => :release,
+      'Profile-staging' => :release,
+      'Profile-production' => :release,
+      'Release' => :release,
+      'Release-staging' => :release,
+      'Release-production' => :release,
+      ...
+    ```
+
+## More information
+
+For more information on creating and using flavors, check out
+the following resources:
+
+* [How to Setup Flutter & Firebase with Multiple Flavors using the FlutterFire CLI][flutterfireCLI]
+* [Build flavors in Flutter (Android and iOS) with different Firebase projects per flavor Flutter Ready to Go][flavors-firebase]
+
+[flutterfireCLI]: https://codewithandrea.com/articles/flutter-firebase-multiple-flavors-flutterfire-cli/
+[flavors-firebase]: {{site.medium}}/@animeshjain/build-flavors-in-flutter-android-and-ios-with-different-firebase-projects-per-flavor-27c5c5dac10b
```

Comment:
Just curious. Why is GitHub showing all of this highlighted text in a brown background color?

Reply:
I noticed that too and wondered. I think it might be because I added this section in a second commit? This is only a guess on my part.